pkgs/servers/http/angie/default.nix

```diff
@@ -3,21 +3,25 @@
 , lib
 , fetchurl
 , nixosTests
+, withAcme ? false
 , withQuic ? false
 , fetchpatch
 , ...
 }@args:
 
 callPackage ../nginx/generic.nix args rec {
-  version = "1.4.1";
+  version = "1.5.1";
   pname = if withQuic then "angieQuic" else "angie";
 
   src = fetchurl {
     url = "https://download.angie.software/files/angie-${version}.tar.gz";
-    hash = "sha256-g6PyuyulnltnZJWiZ01iYG1k6Lz5nO+gneb8M4q3WHo=";
+    hash = "sha256-pSBy+Gv3rSa0WqQOxLHM0V03/vLqvOYBGxrjY9qsBfQ=";
   };
 
-  configureFlags = lib.optional withQuic [
+  configureFlags = lib.optionals withAcme [
+    "--with-http_acme_module"
+    "--http-acme-client-path=/var/lib/nginx/acme"
+  ] ++ lib.optional withQuic [
```

Comment:
```suggestion
  ] ++ lib.optionals withQuic [
```

Reply:
Isn't it recommended to specify `lib.optional` when there is one parameter, and `lib.optionals` when there are several?

Reply:
Not that I have seen so far or know of.

In your changes, you're wrapping twice the value in a list, that's precisely why I request a change.

I wouldn't recommend using lib.optional here.

Reply:
```
nix-repl> :p lib.optional true [ "something" ]
[
  [ "something" ]
]

nix-repl> :p lib.optionals true [ "something" ]
[ "something" ]
```

Reply:
Update.

Reply:
Thank you

Reply:
In Nginx uses `lib.optional` and `lib.optionals` too:
```nix
  ] ++ lib.optionals withPerl [
    "--with-http_perl_module"
    "--with-perl=${perl}/bin/perl"
    "--with-perl_modules_path=lib/perl5"
  ] ++ lib.optional withSlice "--with-http_slice_module"
    ++ lib.optional (gd != null) "--with-http_image_filter_module"
    ++ lib.optional (geoip != null) "--with-http_geoip_module"
    ++ lib.optional (withStream && geoip != null) "--with-stream_geoip_module"
    ++ lib.optional (with stdenv.hostPlatform; isLinux || isFreeBSD) "--with-file-aio"
    ++ configureFlags
    ++ map (mod: "--add-module=${mod.src}") modules;

```

Reply:
That's something I personally don't like but I don't see any double wrapping in there, so I wouldn't request a chance.